stable/database/files/backup_hooks.py

```diff
@@ -142,7 +149,29 @@ def freeze_archive(unfreeze=False):
                 os.kill(int(pid), signal.SIGSTOP)
                 # Make sure all threads are suspended
                 await_suspended(pid)
-    else:
+    elif FREEZE_MODE == "hotsnap":
+        # Freeze or unfreeze the archive using hotsnap
+        sid = processes[0]["sid"]
+        extra_args = []
+        if unfreeze:
+            LOGGER.info("Resuming archiving for nuodb process with startId %s", sid)
```

Comment:
Nit: Can you write the start ID as "start ID"? There is another error or log message where it appears as "start ID".